src/components/ComponentCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+import contentStyles from '../styles/isomer-cms/pages/Content.module.scss';
+
+const ComponentCard = ({
```

Comment:
We can make the component card a little shorter. This is how it looks like in Figma (i think it's too short in the design):
<img width="508" alt="Screenshot 2020-11-04 at 4 49 14 AM" src="https://user-images.githubusercontent.com/31984694/98039423-a68b4600-1e59-11eb-96f6-39543b72ec4c.png">

But if we reduce the height from 240px to 200px it should be fine, this is what it looks like:
<img width="447" alt="Screenshot 2020-11-04 at 4 50 22 AM" src="https://user-images.githubusercontent.com/31984694/98039527-cc184f80-1e59-11eb-995d-52e087200ac5.png">
<img width="452" alt="Screenshot 2020-11-04 at 4 50 55 AM" src="https://user-images.githubusercontent.com/31984694/98039540-ce7aa980-1e59-11eb-886e-2e04cba150a9.png">

Although we can also consider using `rem` instead of `px`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/bc3e5a48d346de942f4dee374a1bcf0001365681